Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-31324__-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2554 @@
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - "Yes"
+      - "No"
+    form:
+      questions:
+      - id: 87805282-9958-404d-a9de-376eea3596bb
+        label: ""
+        labelarg:
+          simple: Please provide the agent IDs for those machines. If you have multiple endpoints, enter them as comma-separated values.
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: shortText
+        options: []
+        optionsarg:
+        - simple: "Yes"
+        - simple: "No"
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Have you find any instances of SAP NetWaver?
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "104":
+    id: "104"
+    taskid: 91541111-9bef-4e5d-a623-5af45152184e
+    type: regular
+    task:
+      id: 91541111-9bef-4e5d-a623-5af45152184e
+      version: -1
+      name: Set Agent IDs to Alert Context
+      description: commands.local.cmd.set.incident
+      script: Builtin|||setAlert
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      agentsid:
+        complex:
+          root: Have you find any instances of SAP NetWaver?.Answers
+          accessor: "0"
+          transformers:
+          - operator: split
+            args:
+              delimiter:
+                value:
+                  simple: ','
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3702.25,
+          "y": 882
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 4961,
+        "width": 3070.25,
+        "x": 2183.25,
+        "y": -465
+      }
+    }
+  }
+inputs:
+- key: SplunkEarliestTime
+  value:
+    simple: -1d@d
+  required: false
+  description: The earliest time for Splunk query.
+  playbookInputQuery:
+- key: SplunkLatestTime
+  value:
+    simple: now
+  required: false
+  description: The latest time for the Splunk search query.
+  playbookInputQuery:
+- key: BlockIndicatorsAutomatically
+  value:
+    simple: "True"
+  required: false
+  description: Whether to block the indicators automatically or not.
+  playbookInputQuery:
+- key: CollectedIndicatorsSeverity
+  value:
+    simple: Malicious
+  required: false
+  description: "The verdict of the collected indicators. Default is \"Malicious\". \nOther options can be \"Suspicious\" and \"Unknown\"."
+  playbookInputQuery:
+- key: RelatedCVEs
```

Comment:
You can remove it and use the ExtractedIndicators.CVE

Reply:
Done